base/_buttons.scss

```diff
@@ -0,0 +1,63 @@
+@charset "UTF-8";
+
+// buttons.scss
+// This file contains all styles related to the button element.
+// see http://bitters.bourbon.io
+// dependent on utilities/color.scss
```

Comment:
Bitters generates a bit of markup, in addition to the needed variables, mixins, functions and placeholders. Most of that markup is concerned with provided sensible typography and base HTML styling, in addition to setting good patterns for later component builds. Maybe this can all be included conditionally in rollups. Let's discuss this further.